src/gazebo_ros_link_attacher.cpp

```diff
@@ -253,6 +257,16 @@ namespace gazebo
       }
       detach_vector.clear();
     }
+
+    joint_state_ = sensor_msgs::JointState();
+    for (const auto& joint : joints) {
```

Comment:
Q: I got a little confused as to where the `joints` variable came from and later realized that it is a private variable. The class seems to have a mixture of private members some with the underscore suffix and some without.

Reply:
yes, it is an external package, thus I don't want to change much. 
However, I don't want to use member variables without the underscore, thus new variables (the ones created by RR developers) do have the underscore.